.pre-commit-config.yaml

```diff
@@ -1,10 +1,37 @@
--   repo: https://github.com/ambv/black
-    rev: stable
+repos:
+  - repo: https://github.com/ambv/black
+    rev: 20.8b1
     hooks:
-    - id: black
-      language_version: python3.7
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.4.0
+      - id: black
+        args: ['--quiet']
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
     hooks:
-    - id: debug-statements
-    - id: flake8
\ No newline at end of file
+      - id: check-ast
+      - id: check-added-large-files
+      - id: check-merge-conflict
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-json
+      - id: check-yaml
+      - id: debug-statements
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: mixed-line-ending
+      - id: trailing-whitespace
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+      - id: flake8
+        additional_dependencies: [
+            'flake8-bugbear',
+            'flake8-comprehensions',
+            'flake8-deprecated',
+            'flake8-print',
+            'flake8-type-checking',
+        ]
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: 'v0.812'
+    hooks:
+      - id: mypy
+        additional_dependencies: [django, djangorestframework, inflection, openapi-spec-validator, prance, pyYAML, django-stubs, djangorestframework-stubs, drf_yasg, drf-spectacular]
```

Comment:
are you sure django&friends are needed here 😅

Reply:
I'm sure they're not! 😄